src/app/PCM/components/pcm-field/non-pcm-field.tsx

```diff
@@ -38,20 +41,34 @@ const NonPcmPcmField: React.FC<NonPcmPcmFieldProps> = (props): JSX.Element => {
           question: ask.question,
           props: config,
         })}
-      {!isEvt && !child && (
-        <Attribute
-          key={code}
-          attribute={code}
-          code={config?.parentCode || ask?.targetCode}
-          size={config?.size}
-          mini={config?.mini}
-          parentCode={config?.parentCode || ask?.targetCode}
-          variant={config?.variant}
-          config={config?.config}
-          styles={config?.styles}
-          hasIndicatorIcon={config?.hasIndicatorIcon}
-        />
-      )}
+      {!isEvt &&
+        !child &&
+        (readonly ? (
+          <Attribute
+            key={code}
+            attribute={code}
+            code={config?.parentCode || ask?.targetCode}
```

Comment:
Rather than checking for the truthiness of config while trying to pass every prop, you could destructure all the props from config and check for it's truthiness there.
ie: let {size, mini, ...rest} = config || {}